productcat/src/main/java/com/visenze/productcat/android/model/DeviceInfo.java

```diff
@@ -87,4 +88,15 @@ public void setUc(String uc) {
         this.uc = uc;
     }
 
+    public void setDoNotTrack(boolean track) {
```

Comment:
as mentioned above, either rename the parameter to setDoNotTrack(boolean doNotTrack) OR create a new method for setTrack(boolean track) to have clearer meaning. This naming convention is a bit misleading if developer did not read the code.